azure/code/train_rapids.py

```diff
@@ -57,7 +57,7 @@ def main():
     print('\n---->>>> cuML version <<<<----\n', cuml.__version__)
 
     compute = 'single-GPU' # 'multi-GPU' option for using multi-GPU algorithms via Dask
-    azure_ml = RapidsCloudML(cloud_type= 'Azure', data_type='Parquet', compute_type=compute)
+    azure_ml = RapidsCloudML(cloud_type = 'Azure', model_type = 'RandomForest', data_type ='Parquet', compute_type = compute)
```

Comment:
This breaks PEP 8, regarding spaces for un-annotated function parameters. Should be func(a=b, c=d)... etc.. to avoid style complaints.